script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1

```diff
@@ -0,0 +1,110 @@
+<Page17>
+"...I’m sorry, I didn’t savor this.
+...Um, could I please have seconds?"
+<Page18>
+"Of course. Please give me a moment, I’ll go refill your bowl."
+With that, she took my bowl and was on her way to the kitchen.
+<Page19>
+And while her back was turned, I spoke up,
+"Ah...Then could you-"
+"? Would you like to request something extra?"
+I tried to ask for what I wanted, but closed my mouth right away. I just couldn’t.
+<Page20>
+"No, never mind. I’ll leave it up to you, Kohaku-san."
+"Yes, by all means. If it comes down to it, I’ll take just a teensy-weensy bit from Akiha-sama’s portions!"
+<Page21>
+"I feel like I don’t want to be around to find out what happens after that..."
+Perhaps feeling elated because of my rare request for seconds, Kohaku-san disappeared into the kitchen.
+<Page22>
+"......But-"
+I’d feel bad if I said this to her, but strangely enough, I still felt somewhat unsatisfied.
+I could feel my appetite’s presence, and the food was delicious too.
+But with every bite I took, my appetite dissipated.
+Like, 'This doesn’t hit the spot' kinda feeling. I craved something less elegantly seasoned and something more... Raw. Like fat, greasy junk.
+<Page23>
+For example, something like raw meat and entrails.
+I could not help but desire something that would remind me of the vividity of life itself, rather than something refined and delicately cooked.
+<Page24>
+With Hisui seeing me off, I leave the Tohno mansion behind me.
+The time is 7:30 AM.
+I did spend some extra time eating, but in the end, I was right back on track at my usual time to school.
+<Page25>
+Monday, the first day of the week. The front of the station is crowded with office workers and students alike, all on their way to their offices and schools respectively.
+This scenery never changes.
+One small thing that has changed though, in front of a certain departmental store, there was a sign that read ‘Entrance Blocked, Do Not Trespass’.
+<Page26>
+"...That store...#"
+Senpai and co. probably cleaned it up in secret.
+A departmental store that ran through a vampire nest.
+Ever since the catacombs collapsed and Vlov died, this place became just another ruin.
+<Page27>
+"...Oh. It all really is over."
+Senpai had said her purpose here was to get rid of the vampires.
+That she’ll leave the city once that’s over.
+So, I can’t ever meet her again.
+There won’t be any more lunchtimes with me, her and Arihiko. No one to talk about random things with. No one to liven up the mood.
+<Page28>
+...To be honest, I have a mountain of regrets.
+Just the thought of Senpai not being there even if I go to school, just makes me want to turn around and go back home.
+Even so, Senpai fought to protect this mundaneness of daily life. To feel so bad about it itself is disrespectful to what she did.
+<Page29>
+Besides, I still remember her.
+'I’ll erase your memories of me tomorrow.', she had said. And yet, she didn’t go through with it.
```

Comment:
> “明日になったら記憶を消す”なんて言っておきながら先輩はそうしなかった。
'Come tomorrow, your memories will be wiped', she said. And yet, she didn’t go through with it.

Despite the phrasing, I don't think this is a direct quote. The closest thing she said was in day 5:
> ですので、貴方の安全を考慮し、ここ一日分の貴方の記憶を消します。いいから抵抗しないように」